### PR TITLE
data: elyanah-aco

### PR DESCRIPTION
**Pull request by elyanah-aco**
## Changes proposed

* Add my profile

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3289"></a>

### CODE REVIEW

File: data/elyanah-aco.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "name": "Connect with me on LinkedIn!",
+      "url": "https://www.linkedin.com/in/elyanahaco2000/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Art Twitter",
+      "url": "https://twitter.com/gising_gising",
+      "icon": "FaTwitter"
+    },
```
Comment (AbbeyIT) [inline data/elyanah-aco.json:23]: The comma should be removed.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

File: data/elyanah-aco.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ],
```
Comment (AbbeyIT) [inline data/elyanah-aco.json:24]: This comma should be removed as well.

Comment (AbbeyIT) [inline data/elyanah-aco.json:24]: This comma should be removed as well.

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

**Review by dan-mba — CHANGES_REQUESTED** [review]
You need a comma at the end of line 6

**Review by dan-mba — APPROVED** [review]
Looks good to me

**Review by AbbeyIT — APPROVED** [review]
LGTM! Thank you for your contribution.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**